src/Client.php

```diff
@@ -273,6 +273,21 @@ public function waitFor(string $cssSelector, int $timeoutInSecond = 30, int $int
         return $this->crawler = $this->createCrawler();
     }
 
+    /**
+     * @throws NoSuchElementException
+     * @throws TimeOutException
+     *
+     * @return Crawler
+     */
+    public function waitForAny(string $cssSelector, int $timeoutInSecond = 30, int $intervalInMillisecond = 250)
```

Comment:
I find the name `waitForAny` a little confusing, particularly when considered in relation to the already-existing `waitFor` method.

`waitFor()` - waits for an element to be rendered
`waitForAny()` - waits for _any_ element to be rendered (incorrect but that was my first guess)

This method waits for an element to be added to the DOM, suggesting:

`waitForElementAddedToDOM()` - unambiguous and more clear but too verbose
`waitForElementPresence()` - possibly better

Reply:
That's right, renamed